doc/parameters/param_list.md

```diff
@@ -429,15 +429,15 @@ When `controller_plugins`\`progress_checker_plugin`\`goal_checker_plugin` parame
 | ----------| --------| ------------|
```

Comment:
https://app.circleci.com/pipelines/github/ros-planning/navigation2/4296/workflows/381da9d4-f80f-46bf-9e61-bf2ad95590f0/jobs/16186 check the CI test jobs, you have some linting errors. You can reproduce these locally with `ament_cpplint` and `ament_uncrustify`